css/style.css

```diff
@@ -46,15 +46,117 @@ Developer @GA in NYC
 /*******************************************/
 
 body {
-     color: #000;
-     font-size: 12px;
-     line-height: 1.4;
-     font-family: Helvetica, Arial, sans-serif;
-     background: url(../images/header_bg.jpg) no-repeat center top rgb(134,141,157);
-      background-size: cover;
+    color: #000;
+    font-size: 12px;
+    line-height: 1.4;
+    font-family: 'Open Sans', sans-serif;
+    background-size: cover;
 }
 
+nav {
+  font-size: 24px;
+  font-weight: 800;
+}
+
+p {
+  font-size: 13px;
+  font-weight: 300;
+}
+
+ul {
+  font-size: 13px;
+  font-weight: 300;
+  list-style: disc;
+  list-style-position: inside;
+  padding: 20px;
+}
+
+li {
+  padding-left: 20px;
+}
+
+blockquote {
+  text-align: center;
+  font-style: italic;
+  font-size: 18px;
+  font-weight: 300 italic;
```

Comment:
this rule is invalid. font-weight can only be set to a number (e.g. `300`) or a predefined value (e.g. `normal`)